nautobot_golden_config/models.py

```diff
@@ -210,6 +224,7 @@ def _get_hierconfig_remediation(obj):
     ComplianceRuleConfigTypeChoice.TYPE_JSON: _get_json_compliance,
     ComplianceRuleConfigTypeChoice.TYPE_XML: _get_xml_compliance,
     RemediationTypeChoice.TYPE_HIERCONFIG: _get_hierconfig_remediation,
+    RemediationTypeChoice.TYPE_DYNAMIC_HIERCONFIG: _get_hierconfig_remediation,
```

Comment:
This is not tracking to me, we are pointing it to the same function of `_get_hierconfig_remediation()`, which would indicated to me, it's the same feature with a different name, but I think I am missing something, can you elaborate on this one?

Reply:
i added that as an option so that if your environment had dynamic functions loaded via git and for a particular remediation setting you didn't need or want to use any dynamic functions at all, it bypassed that loop over the dynamic functions entirely